doc/model_persistence.rst

```diff
@@ -123,3 +123,33 @@ not help in production when performance is critical.
 To convert scikit-learn model to PMML you can use for example `sklearn2pmml
 <https://github.com/jpmml/sklearn2pmml>`_ distributed under the Affero GPLv3
 license.
+
+A more secure format: `skops`
+-----------------------------
+
+`skops <https://skops.readthedocs.io/en/stable/>`__ provides a more secure
```

Comment:
In section 9.1.1, for the first bullet point, we could reference this section to show what is the scope of `skops` (at least for the persistence part).